tests/CMakeLists.txt

```diff
@@ -209,8 +209,14 @@ foreach(test IN ITEMS ${list_all_tests})
 	endif()
 endforeach(test)
 
-build_example_pmemkv_basic_c()
-build_example_pmemkv_basic_cpp()
+if(ENGINE_CMAP)
+	build_example_pmemkv_basic_c()
+	build_example_pmemkv_basic_cpp()
```

Comment:
I think it should be possible to **build** them without any engine compiled-in.

Reply:
yeah, I've got carried away ;)
Done.